src/javascript/Create/CreateNewContentAction/createNewContent.action.jsx

```diff
@@ -4,49 +4,72 @@ import {componentRendererAction, composeActions} from '@jahia/react-material';
 import {reduxAction} from '../../actions/redux.action';
 import {Constants} from '~/ContentEditor.constants';
 import {cmGoto} from '../../ContentManager.redux-actions';
+import {withApolloAction} from './withApolloAction';
+import {from} from 'rxjs';
+import {getActions} from './createNewContent.utits';
 
 const mapDispatchToProps = dispatch => ({
     gotToCE: gotoParams => dispatch(cmGoto(gotoParams))
 });
 
 const mapStateToProps = state => {
     return {
-        uiLang: state.uiLang
+        uiLang: state.uiLang,
+        language: state.language,
+        siteKey: state.site
     };
 };
 
 export default composeActions(
     componentRendererAction,
+    withApolloAction,
     reduxAction(mapStateToProps, mapDispatchToProps),
     {
         init: context => {
-            context.enabled = context.path.startsWith(`/sites/${contextJsParameters.siteKey}/contents`);
+            const variables = {
+                uiLang: context.uiLang,
+                path: context.path,
+                excludedNodeTypes: ['jmix:studioOnly', 'jmix:hiddenType']
+            };
+            context.actions = from(getActions(context, variables));
```

Comment:
Is it the action framework that need to have observable  ?

Reply:
yes, we have to do with it.